.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
```

Comment:
Lets leave Travis out for now if we can't use GitHub actions... Travis is not something we should be setting up for new [projects](https://news.ycombinator.com/item?id=19218036)